panels/templates/schedule/edit.html

```diff
@@ -305,6 +305,8 @@
     };
 
     $(function(){
+        $('#mainContainer').css('display', 'none');
```

Comment:
Highly recommend a comment gets added at some point soon explaining this change -- I can see someone in the future going "Why is this here? Better remove it"